yellowbrick/features/base.py

```diff
@@ -165,6 +177,16 @@ class DataVisualizer(MultiFeatureVisualizer):
         optional string or matplotlib cmap to colorize lines
         Use either color to colorize the lines on a per class basis or
         colormap to color them on a continuous scale.
+        
+    target : str, default: "auto"
```

Comment:
Should we change this argument to `target_type` -- do you think that would be more understandable?

Reply:
Well, this is a matter of choice and conventions. Perhaps, you will know better about this. Just let me know and I will change this.

Reply:
I'm leaning toward `target_type` -- @rebeccabilbro want to way in? @DistrictDataLabs/team-oz-maintainers?

Reply:
I think `target_type` would be much more intuitive to our users (who will probably expect `target == y`)!